### ISSUE TITLE
Incorrect display of certain views

### ISSUE THREAD

**Issue by Cimbali** [mozilla-mobile/mozilla-vpn-client#6370]
**Describe the bug**
Some QML views with a list of items display all items stacked on a single row.

By clicking the home tab and then back to the settings tab, the view is repaired.

**VPN version:**
2.16.0 (built from current master) but I remember it happening from previous builds too.

**Affected Platforms:**
Linux. Currently running OpenSUSE Tumbleweed (rolling release). Latest full dist-upgrade on 13/03/2023. Qt 6 version 6.4.2.

**Prerequisites:**
None?

**Steps to reproduce:**
Open some of the app views that have a list of items. Currently reproducible with excluded apps and privacy settings.

**Expected result:**
All views display properly.

**Actual result:**

Rows of items are stacked on a single row as in these screen shots:
![image](https://user-images.githubusercontent.com/6126377/226398312-75456526-03e8-4d19-862d-2d4bdd455908.png)
![image](https://user-images.githubusercontent.com/6126377/226398424-8c4e6d5f-dded-44d8-89ba-f1dd12b8a065.png)

**Additional notes:**

Many repeated messages like the following appear in the logs:
```log
[20.03.2023 15:57:16.802] Warning: qrc:/ui/screens/settings/appPermissions/AppPermissionsList.qml:82:17: QML VPNCheckBox: Binding loop detected for property "implicitWidth" (AppPermissionsList.qml:82)
```

```log
[20.03.2023 16:00:13.556] Warning: qrc:/nebula/components/VPNCheckBoxRow.qml:24:5: QML VPNCheckBox: Binding loop detected for property "implicitWidth" (VPNCheckBoxRow.qml:24)
```

┆Issue is synchronized with this [Jira Task](https://mozilla-hub.atlassian.net/browse/VPN-4387)

**Comment by Cimbali** [comment]
Actually also happens on the subscription view:

![image](https://user-images.githubusercontent.com/6126377/226426295-376dc211-d7a0-4244-9c1d-1f2fcb47b537.png)

**Comment by birdsarah** [comment]
We have a number of bugs related to this display issue. We currently support Qt 6.4.2. We are currently working on statically linking Qt in linux to prevent these display issues (https://github.com/mozilla-mobile/mozilla-vpn-client/issues/6646). Once that is done we have a workstream to upgrade Qt...but it's probably a Q3 task.

**Comment by Cimbali** [comment]
Unfortunately I have no access to bugs on the jira tracker (such as https://mozilla-hub.atlassian.net/browse/VPN-4494 which seems to be the Ubuntu 23.04 version of this?), which makes following what happens quite hard.

**Comment by Cimbali** [comment]
> We currently support Qt 6.4.2.

Note that when I reported the bug, I was using Qt 6.4.2, so this is happening with the supported version.
I have since updated to Qt 6.5.0 where the same bug happens.

To me, it seems a transition error as by going away and back onto the incorrect view (by clicking the “Home” button in the navigation bar and then the “Settings” button again) the view is corrected.

Fortunately it seems reproducible in the QML inspector so I’ll maybe find something useful there.

**Comment by data-sync-user** [comment]
➤ Valentina Virlics commented:

Verified as fixed on Linux Lunar 23.04 while using Mozilla VPN 2.16.0~rc20230628224117 build.

Postfix ss below.

!lunar issues.png|width=1692,height=527!

!image-20230629-103828.png|width=307,height=616!
